proto/cmp/services/activity/v1alpha1/search_parameters_types.proto

```diff
@@ -2,8 +2,53 @@ syntax = "proto3";
 
 package cmp.services.activity.v1alpha1;
 
+import "cmp/types/v1alpha1/language.proto";
+import "cmp/types/v1alpha1/time.proto";
+import "cmp/types/v1alpha1/price.proto";
+
 // ### Activity Search Parameters
+//These are Activity specific search parameters
```

Comment:
We need an extra space between markdown elements like title and paragraphs. We also need to update the field comments for this.

```suggestion
//
//These are Activity specific search parameters
```